apps/ewallet/lib/ewallet/fetchers/account_fetcher.ex

```diff
@@ -0,0 +1,14 @@
+defmodule EWallet.AccountFetcher do
+  @moduledoc """
+  Module responsible for loading accounts.
+  """
+  alias EWalletDB.User
+
+  def get_highest_account(%{admin_user: admin_user}) do
```

Comment:
Because users can have memberships on separate descendant lines, it's possible that a user has multiple highest accounts. This probably need to handle multiple accounts case.